packages/cactus-common/src/main/typescript/logging/logger.ts

```diff
@@ -21,6 +22,61 @@ export interface ILoggerOptions {
   level?: LogLevelDesc;
 }
 
+/**
+ * STANDARD EXCEPTION HANDLING - EXAMPLE:
+ * ====================================
+ * public doSomething(): void {
```

Comment:
@m-courtin The re-throw here is missing. I want to encourage the reader to abide by the fail-fast principle instead of "log it and forget it".
https://en.wikipedia.org/wiki/Fail-fast

Reply:
@petermetz: Totally agree on the fail-fast principle. Added the re-throw in the documentation of the "Standard Exception Handling" as well as the "Exception Handling with Context"